### PR TITLE
Add default blob loading logic

### PR DESCRIPTION
**Pull request by SwordFaith**

### CODE REVIEW

**Review by debuggy — APPROVED** [review]

File: rest_server/src/models/model_init_handler.js [hunk 1]
```diff
@@ -48,9 +48,27 @@ const createTemplates = async models => {
...
+  try {
+    const envBlob = JSON.parse(process.env.AZURE_STORAGE);
+    const defaultBlob = {
+      type: envBlob.type,
+      storageAccount: envBlob.storage_account,
+      containerName: 'marketplace',
+      connectionStrings: envBlob.connection_strings,
+      tokens: [''],
+      users: [''],
+    };
```
Comment (debuggy) [inline rest_server/src/models/model_init_handler.js:61]: what if the process.env.AZURE_STORAGE is null?

Comment (SwordFaith) [reply rest_server/src/models/model_init_handler.js:61]: It seems json.parse will throw an error "Thrown: SyntaxError: Unexpected token u in JSON at position 0", may be there should be a try catch ?

Comment (debuggy) [reply rest_server/src/models/model_init_handler.js:61]: ok, an overall try catch is enough, no need to add more logics

**Comment by debuggy** [comment]
LGTM :shipit:

**Pull request merged**